packets1/property_message_id_test.go

```diff
@@ -0,0 +1,19 @@
+package packets1
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMessageID(t *testing.T) {
+	msgID := uint16(1234)
+
+	pkt1 := NewPubrec()
+	pkt1.SetMessageID(msgID)
```

Comment:
Maybe assert here that the message ID was set in `pkt1` before continuing? In effect, this would test `SetMessageID()` and increase reader’s confidence about what is going on.

Reply:
I presupposed that `SetMessageID()` is implicitly tested because the value is transferred into `pkg2` but it's imaginable that the value can be put into wrong property and copied from there, so it would be better to explicitly check `MessageID()` value.